src/main/java/gregtech/loaders/recipe/chemistry/NuclearRecipes.java

```diff
@@ -16,22 +35,311 @@ public static void init() {
+
+        BLAST_RECIPES.recipeBuilder().duration(200).EUt(VA[EV]).blastFurnaceTemp(1150)
+                .input(dust, ZirconiumTetrachloride, 5)
+                .input(dust, Magnesium, 2)
+                .output(dust, Zirconium, 1)
+                .output(dust, MagnesiumChloride, 6)
+                .buildAndRegister();
+
+        BLAST_RECIPES.recipeBuilder().duration(200).EUt(VA[EV]).blastFurnaceTemp(1150)
+                .input(dust, HafniumTetrachloride, 5)
+                .input(dust, Magnesium, 2)
+                .output(dust, Hafnium, 1)
+                .output(dust, MagnesiumChloride, 6)
+                .buildAndRegister();
+
+        MIXER_RECIPES.recipeBuilder().duration(200).EUt(VA[EV])
+                .input(dust, Zirconium, 16)
+                .input(dust, Tin, 2)
+                .input(dust, Chrome, 1)
+                .circuitMeta(1)
+                .output(dust, Zircaloy, 19)
+                .buildAndRegister();
+
+        // Inconel 718
+        MIXER_RECIPES.recipeBuilder().duration(200).EUt(VA[EV])
+                .input(dust, Nickel, 5)
+                .input(dust, Chrome, 2)
+                .input(dust, Iron, 2)
+                .input(dust, Niobium)
+                .input(dust, Molybdenum)
+                .circuitMeta(4)
+                .output(dust, Inconel, 11)
+                .buildAndRegister();
+
+        // LEU-235 Dioxide
+        CHEMICAL_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
+                .fluidInputs(HighEnrichedUraniumHexafluoride.getFluid(1000))
+                .fluidInputs(Water.getFluid(2000))
+                .fluidInputs(Hydrogen.getFluid(2000))
+                .output(dust, HighEnrichedUraniumDioxide, 1)
+                .fluidOutputs(HydrofluoricAcid.getFluid(6000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
+                .fluidInputs(LowEnrichedUraniumHexafluoride.getFluid(1000))
+                .fluidInputs(Water.getFluid(2000))
+                .fluidInputs(Hydrogen.getFluid(2000))
+                .output(dust, LowEnrichedUraniumDioxide, 1)
+                .fluidOutputs(HydrofluoricAcid.getFluid(6000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
                 .fluidInputs(DepletedUraniumHexafluoride.getFluid(1000))
-                .output(dust, Uranium238)
-                .fluidOutputs(Fluorine.getFluid(6000))
+                .fluidInputs(Water.getFluid(2000))
+                .fluidInputs(Hydrogen.getFluid(2000))
+                .output(dust, DepletedUraniumDioxide, 3)
+                .fluidOutputs(HydrofluoricAcid.getFluid(6000))
+                .buildAndRegister();
+
+        MIXER_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
+                .input(dust, HighEnrichedUraniumDioxide, 1)
+                .input(dust, DepletedUraniumDioxide, 19)
+                .circuitMeta(1)
+                .output(dust, LEU235, 20)
```

Comment:
high enriched + depleted = low enriched?

Reply:
I think this is according to Zalgo. The main thing that really matters for the LEU235 is that the ratio of Uranium238 and Uranium235 is correct, and as HEU Dioxide is pretty much near 100% U-235, this gets you to the 5% that he preferred. The LEU dioxide, on the other hand, doesn't seem to be assumed to be any particular number, as far as I can tell.

Reply:
Although actually, now that you note it, low enriched uranium dioxide has no uses, so I just did an alternate recipe assuming that it was a little below 20% enrichment.